dearpygui/demo.py

```diff
@@ -43,9 +47,9 @@ def _add_config_options(item, columns, *names, **kwargs):
                 dpg.add_checkbox(label=name, callback=_config, user_data=item, default_value=dpg.get_item_configuration(item)[name])
     else:
         if 'before' in kwargs:
-            dpg.push_container_stack(dpg.add_table(header_row=False, before=kwargs['before']))
+            dpg.push_container_stack(dpg.add_table(header_row=False, before=kwargs['before'], hideable=True))
```

Comment:
Why do we need `hideable=True` here? I believe it doesn't change anything since the headers are hidden, and the code doesn't touch columns anymore. Sort of find/replace mistake?

Same question for all other tables with `headers=False` in this file (there _lots_ of them). These `hideable` arguments seem to contribute half of the changes to `demo.py`, and I bet we can safely get rid of them.

Reply:
Actually, I put all of these `hideable=True` because otherwise it crashes, complaining that there should be that parameter, but I guessed (and I guess) it's the expected behaviour

Reply:
Okay now it's not giving that error anymore... strange... I probably fixed that without noticing it, then I'll remove all of those hideable, thanks!

Reply:
... and now the situation swinged too far and changed to opposite: the PR removes `hideable=True` where the original `demo.py` had it :joy:.

Reply:
Aaah sorry, I have never read it!